resources/monitoring/charts/grafana/dashboards/kubernetes-apiserver-per-client-dashboard.json

```diff
@@ -1,446 +0,0 @@
-{
-  "annotations": {
-    "list": [
```

Comment:
I cannot find this chart in the new directory & on the provisioned cluster for review, is there a replacement for it?

Reply:
is this a custom dashboard?